hackernews/program.py

```diff
@@ -33,20 +35,18 @@ def on_slack_command(event):
         fetch_articles(topic, all_articles)
```

Comment:
can this function just return the list of all articles? why does it need to add to `all_articles`?

Reply:
The function doesn’t fetch all articles because there’s no need to retrieve them all—fetching just one page is sufficient, especially when dealing with a large number of articles (1,000,000+). with a short polling interval it will work fine.